Cognite.Simulator.Utils/Configuration.cs

```diff
@@ -74,11 +75,6 @@ public override void GenerateDefaults()
                 Cognite = new CogniteConfig();
             }
 
-            if (Cognite.CdfRetries == null) {
-                Cognite.CdfRetries.MaxRetries = 11;
-                Cognite.CdfRetries.MaxDelay = 60 * 1000; //ms
-            }
-
```

Comment:
I'm unsure why these lines are removed?

Reply:
We found out these lines were never hit, the Cdfretries values would always default to the ones set here: https://github.com/cognitedata/dotnet-extractor-utils/blob/8b44bc4291d49a28ec6dd3a86f540a0650a6a420/ExtractorUtils/Configuration/BaseConfig.cs#L297

So, we have removed these lines for now.